addon/components/hyper-table/cell.js

```diff
@@ -18,7 +18,7 @@ export default Component.extend({
     let editStatus = this.manager.get('editStatus').find((editStatus) => {
       return editStatus.status === 'editing' || editStatus.status === 'error';
     });
-    if (this.manager.hooks.onRowClicked && !editStatus && !this.selection)
+    if (this.manager.hooks.onRowClicked && !editStatus && !this.selection && this.column.key !== 'rating')
```

Comment:
can you base the condition off an `autosave` attribute on the column or something like that ? same reason as before, the hypertable shouldn't be aware of all that custom `rating` stuff